pkg/ebpf/c/kconfig.h

```diff
@@ -0,0 +1,7 @@
+#ifndef __KCONFIG_H
+#define __KCONFIG_H
+
+#include <linux/kconfig.h>
+#include <asm/compiler.h>
```

Comment:
Would it make sense to have a comment on top of this explaining why we're doing this ?

Reply:
I think it would be a good idea yes

Reply:
Added.